server/pages/index.js

```diff
@@ -162,6 +131,8 @@ function getDefaultContext( request ) {
 	const bodyClasses = [];
 	const cacheKey = getCacheKey( request );
 	const geoLocation = ( request.headers[ 'x-geoip-country-code' ] || '' ).toLowerCase();
+	const isDebug = calypsoEnv === 'development' || request.query.debug !== undefined ? true : false;
```

Comment:
I find those multiple conditions difficult to grok. Is the following not enough?

```javascript
const isDebug = calypsoEnv === 'development' || request.query.debug !== undefined;
```